src/shared/components/instances.tsx

```diff
@@ -27,22 +30,35 @@ export class Instances extends Component<any, any> {
 
     var recommended = [];
     var remaining = [];
+    var values = [];
+
     for (var i of instance_stats.stats.instance_details) {
       if (recommended_instances.indexOf(i.domain) > -1) {
         recommended.push(i);
       } else {
         remaining.push(i);
       }
+
+      // Min number of active users is 10 to include in this calculation
+      if (i.site_info.site_view.counts.users_active_month > 10) {
+        values.push(i.site_info.site_view.counts.users_active_month);
+      }
```

Comment:
Instances with less active users are already filtered out by the crawl script, no need to check it here.